_search-plugins/sparse-search.md

```diff
@@ -1,24 +1,27 @@
 ---
 layout: default
 title: Sparse search
-nav_order: 30
+parent: Search
+nav_order: 50
 has_children: false
-parent: Neural search
+redirect_from:
+  - /search-plugins/neural-sparse-search/
+  - /search-plugins/search-methods/sparse-search/
 ---
 
-# Neural sparse search
+# Sparse search
```

Comment:
We refer to this as both "sparse neural search" and "sparse search" in the following paras. Is that intentional?

Reply:
Changed to "sparse search"